modules/engage-serve-ui-config/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>opencast-engage-serve-ui-config</artifactId>
```

Comment:
Since this is not only for the engage modules but can be used for all user interfaces, let's not call this engage and rather something like `opencast-user-interface-configuration` or `opencast-web-interface-configuration`